python/grizzly/setup.py

```diff
@@ -0,0 +1,42 @@
+import os
```

Comment:
Should we rename this to `pygrizzly` or something, since `grizzly` is taken on PyPI?